### ISSUE TITLE
log scale 0 (integer) breaks the plot

### ISSUE THREAD

**Issue by Moelf** [queryverse/VegaLite.jl#284]
As title says, if 0 presents in the data, we should not draw it, at the moment, it pings everything else to the top instead:
![image](https://user-images.githubusercontent.com/5306213/77225868-a0499300-6b30-11ea-9509-5c9819b6415a.png)

**Comment by davidanthoff** [comment]
Yeah, that is annoying. I needs to be fixed upstream in vega-lite itself. I found this issue https://github.com/vega/vega-lite/issues/3911, not clear what the resolution there was.
